assets/templates/dining_lists/dining_day.html

```diff
@@ -1,37 +1,65 @@
-
 {% extends 'base.html' %}
 {% load staticfiles %}
 {% load humanize %}
 
-{% block title %}Scala Dining {{ date }} {% endblock %}
+{% block title %}Scala Dining {{ date }}{% endblock %}
 
 {% block head %}
     <link rel="stylesheet" href="{% static 'css/dining_v2.css' %}">
 {% endblock %}
 
 {% block content %}
-    <div class="col-12 row p-0 m-0">
-
-        {% if previous_date %}
-            <a href="{% url 'day_view' day=previous_date.day month=previous_date.month year=previous_date.year %}"
-               class="col-2 btn btn-outline-primary">Back</a>
-        {% else %}
-            <a class="col-2"></a>
-        {% endif %}
-
-        <div class="text-center col-8 py-1 h5
-            {% if is_today %} font-bold {% endif %}">
-            <span class="align-middle">{{ date|date|capfirst }}</span>
+    <div class="row">
+        {# Button toolbar #}
+        <div class="col-lg-4">
+            <div class="row">
+                <div class="col-5">
+                    {# (upcoming is a class method, i.e. not specific to the instance) #}
+                    {% with u=date.upcoming %}
+                        <a href="{% url 'day_view' day=u.day month=u.month year=u.year %}"
+                           class="btn btn-block btn-outline-primary">
+                            {% if u == date.today %}Today{% else %}Upcoming{% endif %}
+                        </a>
+                    {% endwith %}
+                </div>
+                <div class="col-7">
+                    <div class="btn-group btn-block">
+                        {% with d=date.previous %}
+                            <a href="{% url 'day_view' day=d.day month=d.month year=d.year %}"
+                               class="btn btn-outline-primary">
+                                <i class="fas fa-chevron-left"></i>
+                                <span class="sr-only">Previous</span>
+                            </a>
+                        {% endwith %}
+                        {% with d=date.next %}
+                            <a href="{% url 'day_view' year=d.year month=d.month day=d.day %}"
+                               class="btn btn-outline-primary">
+                                <i class="fas fa-chevron-right"></i>
+                                <span class="sr-only">Next</span>
+                            </a>
```

Comment:
In the beta trial, someone resereved a slot for 10 years in the future. Thus the reservations of slots has been capped to 30 days to prevent any possible future conflicts. How do we want to represent that in the UI?

Reply:
We could also block access to these days if nothing can happen there anyway (like other comment). That would be my preferable solution for preventing creation of dining slots, I'd be fine with that. I don't really think it's needed though, I expect the users to be good citizens. People could also just claim dining spots and not cook and we also expect them to not do that. If people do these things, so be it, and if it gets really annoying then we'll take care of it